docs/podlet/podlet_to_podlet_communication.md

```diff
@@ -0,0 +1,64 @@
+---
+id: podlet_to_podlet_communication
+title: Podlet To Podlet Communication In The Browser
+---
+
+For situations where 2 or more podlets need to communicate with each other without a page refresh, Podium provides a client side pubsub library called `EventBus`.
```

Comment:
We do not provide a library called `EventBus`

Reply:
true

Reply:
@alexanbj 
You happy with this?:

Podium provides a client side library called `@podium/browser` which provides a pubsub class called `EventBus`.

Reply:
It's better, but it there still no such thing as `EventBus` in that library ;)

Reply:
oh geez, `MessageBus`.